### ISSUE TITLE
print error-message inside sonic-cfggen if any stderr is masked

### ISSUE THREAD

**Issue by kannankvs** [Azure/sonic-buildimage#3755]
This issue is based on the [issue3527](https://github.com/Azure/sonic-buildimage/issues/3527) and the feedback from related [PR3745](https://github.com/Azure/sonic-buildimage/pull/3745).
Feedback in PR says "can you print error message in sonic-cfggen saying the data[yyy] is not available to stderr? This can be a separate pr".
This issue is opened to track this requirement.

**Description**  
When device boots up, NTP module calls sonic-cfggen to read a management vrf configuration which returns a traceback if the configuration does not exist. If the particular configuration does not exist, NTP starts in default VRF after ignoring the error, which is fine. The traceback print is unnecessary and hence the PR3745 masks the stderr returned by sonic-cfggen.
When this stderr is masked, the new requirement is to print error message in sonic-cfggen saying the data[yyy] is not available to stderr.

**Steps to reproduce the issue:**  
1. Reboot with master image and check "journalctl -b --no-pager > j1" and open the file to search for MGMT_VRF_CONFIG traceback from NTP. Once if PR3745 is merged, this traceback will not be visible. 

**Describe the results you received:**  
The traceback that was seen before merging PR3745 is no more available. i.e. stderr returned by sonic-cfggen is masked while calling it from "ntp" script.

**Describe the results you expected:**  
sonic-cfggen has to print that the stderr is masked.

    **Output of `show version`:**
Master image
